src/cms/views/feedback/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Python standard Init-File
+"""
```

Comment:
```suggestion
```
This is not an informative docstring. Either explain the purpose of the python module here or just omit the docstring completely.